### DISCUSSION TITLE
Different folder for Upscaled images?

### DISCUSSION THREAD

**Discussion by E2GO** [lllyasviel/Fooocus#1974] [Q&A]
Hey! So the question is - is there a way to use different folder for upscaled images? Thanks in advance)

**Comment by mashb1t** [comment]
You mean to put 1.5 to a 1.5x folder and 2x to another?
This does not exist OOTB but feel free to add a parameter to the log() method of private_logger.py and the calls in async_worker.py to use the uov_method as part of the path if this is set.

**Reply by E2GO** [reply]
Not for each kind of upscale but for all upscales. Thanks for guide - will try to check it
